Week1/homework/js-exercises/ex9.js

```diff
@@ -0,0 +1,13 @@
+//Exercise 9: Log the remainder
+//x after the calculation
+let x = 7;
+let calculationX = x % 3;
```

Comment:
While `calculationX` is a descpriptive name it could be a little bit more specific.

Reply:
i don't understand what you mean but i change all the let to const :)